### PR TITLE
W01 Individual Activity: Add Item to Cart Broken, completed.

### PR DESCRIPTION
**Pull request by Juarpla**
- Change the type of data structure where products will be save in local storage.
- Fixed in Json data base a path for a image.

### CODE REVIEW

File: src/index.html
Comment (carrotjived) [inline src/index.html:1]: Good

**Review by carrotjived — APPROVED** [review]
The bug is fixed, and we have the same code in the ADd to cart. Corrected the image path for one of the items

File: src/json/tents.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "Id": "989CG",
     "NameWithoutBrand": "Talus Tent - 3-Person, 3-Season",
     "Name": "The North Face Talus Tent - 3-Person, 3-Season",
-    "Image": "../images/tents/the-north-face-talus-tent-3-person-3-season-in-golden-oak-saffron-yellow~p~989cg_01~320.jpg",
+    "Image": "../images/tents/the-north-face-talus-tent-4-person-3-season-in-golden-oak-saffron-yellow~p~985rf_01~320.jpg",
```
Comment (Julirey) [inline src/json/tents.json:51]: Although the image path correction is good, the information of the image filename doesn't match the information of the JSON data.

File: team.txt [hunk 2]
```diff
@@ -3,4 +3,8 @@ Team 19 Sleep Outside Trello Board:
 
 Team 19 Members:
     -> Victor Onato
-    -> Juan Plasencia
\ No newline at end of file
+    -> Juan Plasencia
+    -> Brandon Cushenberry
+    -> Claudia Esparza
+    -> Julio Reyes
```
Comment (Julirey) [inline team.txt:9]: That's me 👍

**Review by Julirey — COMMENTED** [review]
Don't mind me, I know the branch is closed. I'm trying out the review process.

**Pull request merged**